fbgemm_gpu/experimental/gen_ai/src/comm/car.cpp

```diff
@@ -272,7 +272,13 @@ void nccl_allreduce(
     default:
       TORCH_CHECK(false, "unsupported type: ", src.scalar_type());
   }
-#if defined(USE_ROCM)
+
+// ncclAllReduceWithBias landed in OSS RCCL only recently, so it won't be
+// available in public ROCm for a while.  As such, its use has to be guarded by
+// a far-off ROCm version check.
+//    https://github.com/ROCm/rccl/pull/1729
+#if (defined(FBGEMM_FBCODE) && defined(USE_ROCM)) || \
+    (!defined(FBGEMM_FBCODE) && defined(USE_ROCM) && (ROCM_VERSION >= 60600))
```

Comment:
@q10, I believe the ROCM_VERSION guard here should be higher. Maybe 70000? I am on 6.4 right now and `ncclAllReduceWithBias` is still not supported.